docs/sdks/server-sdks/node.md

```diff
@@ -104,8 +80,26 @@ More details about logging and examples (with Segment, Rudderstack, mParticle, a
 
 Eppo's SDK uses an internal cache to ensure that duplicate assignment events are not logged to the data warehouse. While Eppo's analytic engine will automatically deduplicate assignment records, this internal cache prevents firing unnecessary events and can help minimize costs associated with event logging. 
 
+## 3. Initialize the SDK
+
+Initialize the SDK with a SDK key, which can be generated in the Eppo interface. Initialization should happen when your application starts up to generate a singleton client instance, once per application lifecycle:
+
+```javascript
+import { init } from "@eppo/node-server-sdk";
+
+await init({
+  apiKey: "<API_KEY>",
```

Comment:
```suggestion
  apiKey: "<SDK_KEY>",
```
It's unfortunate that the code calls it `api_key` instead of `sdk_key`, but maybe this would make it a bit clearer?

Reply:
updated